app/reducers/contactsform.js

```diff
@@ -270,7 +267,7 @@ contactFormSelectors.manualFormIsValid = createSelector(manualSearchQuerySelecto
   }
   return {
     errors,
-    isValid: isEmpty(errors)
+    isValid: errors.length === 0
```

Comment:
Errors is an object, has no length method. `Object.keys(errors).length === 0` would work.